web-common/src/features/workspaces/MetricsWorkspace.svelte

```diff
@@ -117,49 +77,51 @@
       instanceId,
       e.currentTarget,
       filePath,
-      metricViewName,
+      fileName,
       fileArtifacts.getNamesForKind(ResourceKind.MetricsView),
     );
     if (newRoute) await goto(newRoute);
   }
 </script>
 
-<svelte:head>
-  <title>Rill Developer | {fileName}</title>
-</svelte:head>
-
-{#if fileNotFound}
-  <WorkspaceError message="File not found." />
-{:else}
-  <WorkspaceContainer inspector={isModelingSupported}>
-    <WorkspaceHeader
-      slot="header"
-      showInspectorToggle={isModelingSupported}
-      titleInput={fileName}
-      on:change={onChangeCallback}
-    >
-      <div slot="cta" class="flex gap-x-2">
-        <Tooltip distance={8}>
-          <Button on:click={() => (showDeployModal = true)} type="secondary">
-            Deploy
-          </Button>
-          <TooltipContent slot="tooltip-content">
-            Deploy this dashboard to Rill Cloud
-          </TooltipContent>
-        </Tooltip>
-        <PreviewButton
-          dashboardName={metricViewName}
-          status={previewStatus}
-          disabled={previewDisabled}
-        />
-      </div>
-    </WorkspaceHeader>
-
-    <MetricsEditor slot="body" {yaml} {filePath} {allErrors} {metricViewName} />
-
-    <MetricsInspector {filePath} slot="inspector" />
-  </WorkspaceContainer>
-{/if}
+<svelte:window on:focus={() => fileArtifact.fetchContent(true)} />
```

Comment:
Looks like this should apply to all workspaces?

Reply:
I don't really have a preference, but, just to confirm, this would be a behavior change. If  you look at the codebase now, the only place where `refetchOnWindowFocus` is true is the dashboard editor page. I was thinking about moving this to the Editor component and having it be configurable via a prop. Is there any file type this shouldn't apply to?

Reply:
Oh, got it. Seems like it should apply to all workspaces, as long as unchanged files don't trigger needless ingest and profiling on the Source and Model workspaces.